Health App/Home/HealthConcernsListVC.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  HealthConcernsListVC.swift
+//  Health App
+//
+//  Created by Koushik Dutta on 07/10/17.
+//  Copyright © 2017 Virtual Elf LLC. All rights reserved.
+//
+
+import UIKit
+import DataLayer
+
+class HealthConcernsListVC: UIViewController {
+    
+    var inControlStatus: [HealthConcern] = []
```

Comment:
`var inControlStatus : [HealthConcern] {
return DatabaseManager.shared.fetchHealthConcernsBasedOnStatus(status: .inControl)
}`

take `DatabaseManager.HealthConcernStatusType` as parameter type and `not string`

Reply:
use computed properties ... no need to `fetchBasedOnStatus()` everytime